thinklmi-user/thinklmi.c

```diff
@@ -39,7 +39,7 @@ void get_settings_all(int fd)
         perror("query_apps ioctl get");
     } else {
 	printf("Total settings: %d\n", settings_count);
-	for(i=0; i <= settings_count; i++)
+	for(i=0; i <= 0xFF; i++)
```

Comment:
I really don't like this - why is this being done?

Reply:
replaced with TLMI_MAX_SETTINGS
The settings count was missing some of the settings at the end.
So we have to parse through the entire settings and display the valid ones

Reply:
Gotcha - this is better.

Can you please change higher in the file where settings is declared to also use this define
	unsigned char *settings[256];
To keep everything consistent :)

Reply:
Done